README.md

```diff
@@ -27,10 +27,14 @@ Axon is a high-performance layer2 framework with native cross-chain function. B
 
 Axon is compatible with [Ethereum](https://ethereum.org) so that all of the develop utilities can be used on it directly. And the [Axon CLI](https://github.com/axonweb3/axon-cli) provides an all-in-one client which includes initialization, DevOps, cross-chain request and so on. Seeing is believing, there is a [15 minutes tutorial](https://docs.axonweb3.io/getting-started/for-dapp-devs/zero_to_axon_with_axon_cli) that will lead you to build your own chain and deploy a tiny application.
```

Comment:
Axon, an Ethereum(https://ethereum.org) compatible chain, use all develop utilities **(_not sure about this word)**_  in Ethereum has. Also, [Axon CLI](https://github.com/axonweb3/axon-cli), provides an all-in-one client including initialization, DevOps, cross-chain request and so on.  Seeing is believing, here is a [15 minutes tutorial](https://docs.axonweb3.io/getting-started/for-dapp-devs/zero_to_axon_with_axon_cli) provides hands-on experience of building customized chains and deploying applications.